app/lib/country_code.rb

```diff
@@ -29,9 +29,8 @@ def european_economic_area?(code)
     end
 
     def secondary_education_teaching_qualification_required?(code)
```

Comment:
I think we can remove this method now and replace the two places it's used with direct access to the `country`, to avoid an unnecessary database query in this method.

Reply:
I've made all the other changes suggested however I'm getting some errors with this I can't resolve.

Reply:
I was able to replace the `qualified_for_subject_required?` method with:

```rb
Country.exists?(code: country_code, subject_limited: true)
```

What errors were you getting?

Reply:
That's working for me without error now